x/datapool/types/message_pool.go

```diff
@@ -48,6 +48,24 @@ func (msg *MsgRegisterDataValidator) GetSigners() []sdk.AccAddress {
 
 var _ sdk.Msg = &MsgCreatePool{}
 
+func NewMsgCreatePool(poolParams *PoolParams, curator string) *MsgCreatePool {
+	params := &PoolParams{
+		DataSchema:            poolParams.DataSchema,
+		TargetNumData:         poolParams.TargetNumData,
+		MaxNftSupply:          poolParams.MaxNftSupply,
+		NftPrice:              poolParams.NftPrice,
+		TrustedDataValidators: poolParams.TrustedDataValidators,
+		TrustedDataIssuers:    poolParams.TrustedDataIssuers,
+		Deposit:               poolParams.Deposit,
+		DownloadPeriod:        poolParams.DownloadPeriod,
+	}
+
+	return &MsgCreatePool{
+		Curator:    curator,
+		PoolParams: params,
+	}
```

Comment:
I think it can be changed like this.
Is there any reason to declare this new?
```suggestion
	return &MsgCreatePool{
		Curator:    curator,
		PoolParams: poolParams,
	}
```

Reply:
Oh, there is no reason for this. Thanks. I'll refactor it

Reply:
modified in [5f69ad0](https://github.com/medibloc/panacea-core/pull/287/commits/5f69ad06762947b77e90421a4b2487f058772fa2)